readme.md

```diff
@@ -0,0 +1,148 @@
+# Forms
+
+A set of utilities to help you create forms in React.
+
+## Motivation
+
+Creating ad-hoc forms using React hooks is quite easy, but there is one thing that is a bit hard to
+do: preventing to render the complete form on each keystroke.
+
+Another motivation is to make form handling within our applications more consistent.
+
+## Installation
+
+```
+yarn add @kaliber/forms
+```
+
+## Usage
```

Comment:
Ik denk dat het goed is om ook een kopje ### Hooks toe te voegen. Om de basis werking uit te leggen, maar ook vooral de dingen die je misschien niet direct zou verwachten maar wel heel handig zijn: het meegeven van `form` aan `useFieldSnapshot(form)` bijvoorbeeld.

Reply:
Goed idee!